### DISCUSSION TITLE
I have a problem installing Windows when using the file

### DISCUSSION THREAD

**Discussion by 7HRAMZI** [memstechtips/UnattendedWinstall#151] [Q&A]
Hi

I need help

I am having trouble using autounattend file and including or putting it in Windows installation file. I get error or something similar.
The version that is not working is Windows 10 Pro 64 bit, but I don't know Windows 11 and I haven't tried but the version that is not working is Windows 10 Pro 64 bit
I used NTLite to edit Windows version and it worked to boot the system but without integrating autounattend file but when I merge autounattend file I get error message
I don't have technical expertise in this field

![Capdture](https://github.com/user-attachments/assets/087c02d9-e12c-4751-a1d7-647787355162)

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
